python/social_bot/envs/cartpole.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+import gym
+import os
+import logging
+import numpy as np
+import random
+import math
+
+from gym import spaces
+import social_bot
+from social_bot import teacher
+from social_bot.teacher import TeacherAction
+import social_bot.pygazebo as gazebo
+
+logger = logging.getLogger(__name__)
+
+
+class CartPole(gym.Env):
+    """
+    This environment simulates the classic cartpole in the pygazebo environment.
+    """
+
+    def __init__(self,
+                 max_steps=100,
+                 x_threshold=5,
+                 theta_threshold=0.314,
+                 noise=0.001):
```

Comment:
comments about x_threshold, theta_threshold and noise

Reply:
Also it's better to comment the meaning of each dimension of the action, the observation, how reward is given and when one episode ends.